tests/integrations/test_dsm.py

```diff
@@ -66,16 +66,24 @@ def setup_dsm_rabbitmq(self):
     def test_dsm_rabbitmq(self):
         assert self.r.text == "ok"
 
+        if context.library == "nodejs":
```

Comment:
could we add a comment about where these hashes come from and why nodejs is different?

Reply:
Added!